From 1674147a149c2165a927a5d8eb0db4eee1f6a4e3 Mon Sep 17 00:00:00 2001
From: Thierry Reding <treding@nvidia.com>
Date: Tue, 5 Jan 2016 15:21:23 +0100
Subject: [PATCH 015/117] tests: util: Fixup util_open() parameter order

util_open() takes a device parameter, followed by a module parameter.
The existing tests used the drmOpen() function, which uses a different
ordering of the parameters, and the old ordering was accidentally kept
during the conversion.

Signed-off-by: Thierry Reding <treding@nvidia.com>
Reviewed-by: Emil Velikov <emil.l.velikov@gmail.com>
---
 tests/modetest/modetest.c | 2 +-
 tests/proptest/proptest.c | 2 +-
 tests/vbltest/vbltest.c   | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/modetest/modetest.c b/tests/modetest/modetest.c
index 22e3e81..f665240 100644
--- a/tests/modetest/modetest.c
+++ b/tests/modetest/modetest.c
@@ -1603,7 +1603,7 @@ int main(int argc, char **argv)
 	if (!args)
 		encoders = connectors = crtcs = planes = framebuffers = 1;
 
-	dev.fd = util_open(module, device);
+	dev.fd = util_open(device, module);
 	if (dev.fd < 0)
 		return -1;
 
diff --git a/tests/proptest/proptest.c b/tests/proptest/proptest.c
index 24c6345..4bd0866 100644
--- a/tests/proptest/proptest.c
+++ b/tests/proptest/proptest.c
@@ -295,7 +295,7 @@ int main(int argc, char *argv[])
 
 	args = argc - optind;
 
-	fd = util_open(module, device);
+	fd = util_open(device, module);
 	if (fd < 0)
 		return 1;
 
diff --git a/tests/vbltest/vbltest.c b/tests/vbltest/vbltest.c
index 1833321..4475b49 100644
--- a/tests/vbltest/vbltest.c
+++ b/tests/vbltest/vbltest.c
@@ -120,7 +120,7 @@ int main(int argc, char **argv)
 		}
 	}
 
-	fd = util_open(module, device);
+	fd = util_open(device, module);
 	if (fd < 0)
 		return 1;
 
-- 
2.7.4

